saleor/shipping/utils.py

```diff
@@ -53,3 +54,17 @@ def convert_to_shipping_method_data(
         minimum_order_price=minimum_order_price,
         maximum_order_price=maximum_order_price,
     )
+
+
+def annotate_active_shipping_methods(
+    shipping_methods: List["ShippingMethodData"],
+    excluded_methods: List["ExcludedShippingMethod"],
+):
+    reason_map = {str(method.id): method.reason for method in excluded_methods}
+    for instance in shipping_methods:
+        instance.active = True
```

Comment:
I am confusing, method name tells me that we annotate object with something but actually we are changing `active` state either.

Reply:
Agree, looks like this function just update the status of all shipping methods based on the list of excluded shipping methods, recieved from webhook.
Can we change the name?

Reply:
I used the verb `initialize` instead in order to suggest that accessing this field makes no sense prior to initialisation